### ISSUE TITLE
stylelint 12.x as peer dependency

### ISSUE THREAD

**Issue by ybiquitous** [YozhikM/stylelint-a11y#45]
Hi,

stylelint [12.0.0](https://github.com/stylelint/stylelint/releases/tag/12.0.0) was released but do you have a plan to update the peer dependency?

https://github.com/YozhikM/stylelint-a11y/blob/1c522f677e1bd68420cc47a935e9865dd797159a/package.json#L40

I get the following error when installing `stylelint@12` and `stylelint-a11y`. 😢 

> npm ERR! peer dep missing: stylelint@^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0, required by stylelint-a11y@1.2.2

It would be great if you could consider it.
Thanks.

**Comment by ybiquitous** [comment]
@YozhikM Thank you so much for the new release 1.2.3! 🎉 
https://github.com/YozhikM/stylelint-a11y/releases/tag/1.2.3
